File: Up to Week 7/C/pset1/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void print_row(int bricks);

int main(void)
{
    int height;
    // Prompt user for input
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
    for (int i = 0; i < height; i++)
    {
        // Print spaces
        int diff = height - i - 1;
        for (int j = 0; j < diff; j++)
        {
            printf(" ");
        }
        // Print left hashes
        print_row(i + 1);

        // Print gap
        printf("  ");

        // Print right hashes
        print_row(i + 1);
        printf("\n");
    }
}

void print_row(int bricks)
{
    for (int i = 0; i < bricks; i++)
    {
        printf("#");
    }
}

File: Up to Week 7/Python/pset6/sentimental-mario-more/mario.py
from cs50 import get_int

# Get user input
while True:
    height = get_int("Height: ")
    if height <= 8 and height >= 1:
        break


row = 0
# Print for each row
for i in range(height):
    print(" " * (height - row - 1), end="")
# Print Left Hash
    for i in range(row + 1):
        print("#", end="")
    # Print space
    print("  ", end="")
    # Print right Hash
    for i in range(row + 1):
        print("#", end="")
    print()
    row += 1
